test/issue1180-local-cache-broken/dub.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "test",
+	"dependencies": {
+		"local-cache-lib": "~master"
```

Comment:
It is hard to answer here, because I do not know whether --cache=local is fully implemented and works as designed or a half implemented feature. There is no documentation.

Yes, as the depencenies(fetched with cache local) stored in the package folder, this could conflict with sub modules. My idea is to enhance the cache=local functionality and store the depencenies in a subfolder "packages". 

But maybe there is a reason why cache=local works as it works. Is the idea that the developer adds the path by hand to dub json?